examples/visual/car_sequencing.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ongoing-target",
+   "metadata": {},
+   "source": [
+    "# Car Sequencing problem in CPMPy, with visualization\n",
+    "\n",
+    "Based on Alexander Schiendorfer's, but with simplified visualization\n",
+    "https://github.com/Alexander-Schiendorfer/cp-examples/tree/main/car-sequencing\n",
+    "\n",
+    "Given different types of cars (based on the present options), an overall car demand per type \n",
+    "and some constraints on how many times properties can be scheduled in consecutive timeslots,\n",
+    "the program finds a feasible sequencing of the different cars for a timetable.\n",
+    "\n",
+    "This problem is also described in CSPlib: https://www.csplib.org/Problems/prob001/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "universal-extent",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Academic license - for non-commercial use only - expires 2022-10-21\n",
+      "Using license file /home/tias/local/src/gurobi912/linux64/../gurobi.lic\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.colors as mcolors\n",
+    "from cpmpy import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "worldwide-spell",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the CPMpy model and variables\n",
+    "def model_car_sequence(demand, per_slots, at_most, requires):\n",
+    "    nSlots = sum(demand)  # The amount of timeslots to be filled\n",
+    "    nOptions = len(at_most)  # The amount of different options\n",
+    "    nCarConfigs = len(demand)  # The amount of different car types\n",
+    "    requires = cpm_array(requires)  # For element constraint\n",
+    "\n",
+    "    # Decision variables\n",
+    "    slots = intvar(1, nCarConfigs, shape=nSlots, name=\"slots\")  # Sequence of different car types\n",
```

Comment:
Writing it 0-indexed might be a little clearer in the constraint themselves? But 1-indexed seems the way it is done most of the time indeed....